app/lib/homework/homework_dialog/homework_dialog_bloc.dart

```diff
@@ -675,6 +791,16 @@ class HomeworkDialogBloc extends Bloc<HomeworkDialogEvent, HomeworkDialogState>
     );
   }
 
+  Date _getNextSchoolday() {
+    final today = _clock.now().toDate();
+    final daysUntilNextSchoolday = switch (today.weekDayEnum) {
+      WeekDay.friday => 3, // Monday
+      WeekDay.saturday => 2, // Monday
+      _ => 1 // Tomorrow
+    };
+    return today.addDays(daysUntilNextSchoolday);
```

Comment:
Things we do later (someday in the future):

* Add holidays to the calculation
* We assume that Monday starts the school, we could also pull the activated weekdays from the timetable feature ("Einstellungen" -> "Stundenplan" -> "Aktivierte Wochentage"). In case someone doesn't have school on Monday, it would take Tuesday as the next school day.

Reply:
I add these as issues #1179 #1180